app/modules/tmr.c

```diff
@@ -151,6 +151,7 @@ static timer_t tmr_get( lua_State *L, int stack ) {
 	if (lua_isuserdata(L, stack)) {
 		return (timer_t)luaL_checkudata(L, stack, "tmr.timer");
 	} else {
+		platform_print_deprecation_note("Use the dynamic timer API rather than the static one.", "end of 2016");
```

Comment:
@marcelstoer if there can be a macro, maybe good resolution is __FUNC__ / __FILE__ pseudo-variable can be used. So don't remember how them exactly named, but in macro-way there are options.